spectrogram-feature-extraction.html

```diff
@@ -0,0 +1,468 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>Spectrogram feature extraction - iodide</title>
+<link rel="stylesheet" type="text/css" href="https://iodide.io/master/iodide.master.css">
+</head>
+<body>
+<script id="jsmd" type="text/jsmd">
+%% meta
+{
+  "title": "Spectrogram feature extraction",
+  "viewMode": "presentation",
+  "lastSaved": "2018-06-28T09:42:27.565Z",
+  "languages": {
+    "js": {
+      "pluginType": "language",
+      "languageId": "js",
+      "displayName": "Javascript",
+      "codeMirrorMode": "javascript",
+      "module": "window",
+      "evaluator": "eval",
+      "keybinding": "j",
+      "url": ""
+    },
+    "py": {
+      "languageId": "py",
+      "displayName": "python",
+      "codeMirrorMode": "python",
+      "keybinding": "p",
+      "url": "https://iodide.io/pyodide-demo/pyodide.js",
+      "module": "pyodide",
+      "evaluator": "runPython",
+      "pluginType": "language"
+    }
+  },
+  "lastExport": "2018-06-28T10:04:40.874Z"
+}
+
+%% md
+# Spectrogram Feature Extraction
+
+In this experiment, we'll attempt to extract a spectrogram feature from a .wav audio file encoded as pcm 16 and then we will record our own audio to extract the spectrogram
+
+To do this we will only use javascript cells to demonstrate the pyodide API and how you could apply this same code to any project
+
+%% md
+## Loading Python
+
+The first thing we want to do is load the Pyodide plugin for Iodide, which will
+give us Python support. (Don't worry too much about the details here. This
+is just a snippet of JSON that describes how to load and use the language
+plugin.)
+
+%% plugin
+{
+  "languageId": "py",
+  "displayName": "python",
+  "codeMirrorMode": "python",
+  "keybinding": "p",
+  "url": "https://iodide.io/pyodide-demo/pyodide.js",
+  "module": "pyodide",
+  "evaluator": "runPython",
+  "pluginType": "language"
+}
+
+%% md
+We now have a Python environment loaded and ready to go.  Let's test it.
+
+%% js
+pyodide.runPython('import sys\nsys.version')
```

Comment:
Is there a reason to use `runPython` rather than a Python cell here?  I think it makes things much less legible.

Reply:
as i said in the introduction
`To do this we will only use javascript cells to demonstrate the pyodide API and how you could apply this same code to any project`
i wanted to use only js cells to demonstrate the pyodide API and make it easy for anyone to port this code to a web app

Reply:
i'll leave it there for now, but if you really want me to change, i can do so